src/components/FilterList/FilterLIst.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+
+import cn from 'classnames';
+
+type Props = {
+  itemCount: number,
+};
+
+export const FilterList: React.FC<Props> = ({ itemCount }) => {
+  const [filterBy, setFilterBy] = useState('All');
+
+  const changeFilter = (newValue: string) => {
+    setFilterBy(newValue);
+  };
+
+  return (
+    <>
+      <span className="todo-count" data-cy="todosCounter">
+        {`${itemCount} `}
+        items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={cn('filter__link', { selected: filterBy === 'All' })}
+          onClick={() => {
+            changeFilter('All');
+          }}
```

Comment:
![image](https://user-images.githubusercontent.com/96409563/210001793-8580b4a9-a3ac-4623-969e-31c72680aed6.png)
Would be even better to create an [Enum ](https://www.typescriptlang.org/docs/handbook/enums.html)
Also, let's create separate functions to handle the clicks